# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/base/nsSocketTransport2.h
# Commit: 8e7c97a37073
# Full Hash: 8e7c97a37073a14ba832e2f5c27f2c11b7787ef5
# Author: Valentin Gosu <valentin.gosu@gmail.com>
# Date: 2023-03-15 16:26:49
# Regressor Bug: 1805372
# File Overlap Count: 1
# Description:
#   Bug 1805372 - Pass effectiveTRRMode to nsHttpChannel from socketTransport/DnsAndConnectSocket r=necko-reviewers,kershaw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D164539
# ==============================================================================

diff -r 5abfd5ad2e35 -r 8e7c97a37073 netwerk/base/nsSocketTransport2.h
--- a/netwerk/base/nsSocketTransport2.h	Wed Mar 15 12:41:26 2023 +0000
+++ b/netwerk/base/nsSocketTransport2.h	Wed Mar 15 13:00:46 2023 +0000
@@ -326,6 +326,8 @@
   nsCString mEchConfig;
   bool mEchConfigUsed = false;
   bool mResolvedByTRR{false};
+  nsIRequest::TRRMode mEffectiveTRRMode{nsIRequest::TRR_DEFAULT_MODE};
+  nsITRRSkipReason::value mTRRSkipReason{nsITRRSkipReason::TRR_UNSET};
 
   // mNetAddr/mSelfAddr is valid from GetPeerAddr()/GetSelfAddr() once we have
   // reached STATE_TRANSFERRING. It must not change after that.